gbus/abstractions.go

```diff
@@ -192,6 +192,7 @@ type Invocation interface {
 	Tx() *sql.Tx
 	Ctx() context.Context
 	Routing() (exchange, routingKey string)
+	RetryInfo() (attempt, maxRetryCount uint)
```

Comment:
I think it would be better to have this renamed to DeliveryInfo which returns a struct containing the attempt and maxAttemptCount fields.
That way we can put on it more data later on without changing the interface